sql/2024/accessibility/a11y_technology_usage_by_domain_rank.sql

```diff
@@ -0,0 +1,26 @@
+#standardSQL
+# A11Y technology usage by domain rank
+SELECT
+  client,  # Client domain
+  rank_grouping,  # Grouping of domains by their rank
+  COUNT(DISTINCT page) AS total_in_rank,  # Total number of unique sites within the rank grouping
+  tech.technology AS app,  # Accessibility technology used
+  COUNT(DISTINCT IF(category = 'Accessibility', page, NULL)) AS sites_with_app,  # Number of sites using the specific accessibility technology
+  COUNT(DISTINCT IF(category = 'Accessibility', page, NULL)) / COUNT(DISTINCT page) AS pct_sites_with_app  # Percentage of sites using the accessibility technology
+FROM
+  `httparchive.all.pages`,
+  UNNEST(technologies) AS tech,  # Expand technologies array to individual rows
+  UNNEST(categories) AS category,  # Expand categories array to individual rows
+  UNNEST([1000, 10000, 100000, 1000000, 10000000]) AS rank_grouping  # Expand rank_grouping to cover different rank categories
```

Comment:
We added a new category here in last year:

```suggestion
  UNNEST([1000, 10000, 100000, 1000000, 10000000, 100000000]) AS rank_grouping  # Expand rank_grouping to cover different rank categories
```